index.html

```diff
@@ -8,30 +8,236 @@
                     don’t hestiate to contact me.
-                </p>
-                <address>
-                    <ul class="container nav-container">
-                        <li class="nav-link-container"><a href="github.com/fobadara" class="nav-link"><img src="images/github.svg" alt="An image showing Github's logo"></a></li>
-                        <li class="nav-link-container"><a href="#" class="nav-link"><img src="images/linkedin-icon.svg" alt="An image showing Linkedin's logo"></a></li>
-                        <li class="nav-link-container"><a href="#" class="nav-link"><img src="images/angellist.svg" alt="An image showing angelist's logo"></a></li>
-                        <li class="nav-link-container"><a href="#" class="nav-link"><img src="images/twitter.svg" alt="An image showing twitter's logo"></a></li>
-                        <li class="nav-link-container"><a href="#" class="nav-link"><img src="images/medium.svg" alt="An image showing medium's logo"></a></li>
+                </div>
+                <div class="btn-container" id="resume-btn-container">
+                    <button type="button" class="btn" id="resume-btn">Get My Resume</button>
+                </div>
+            </section>     
+        </aside>
+        <div class="skillset">
+            <article class="all-skills">
+                <div class="box-container" id="lang-container">
+                    <div class="logo-container" id="lang-logo-container">
+                        <img src="images/lang.svg" alt="" class="type" id="lang-logo">
+                    </div>
+                    <div class="header-container">
+                        <h3 class="header" id="lang">Languages</h3>
+                    </div>
+                    <ul class="container code-skill" id="code-lang-list">
+                        <li class="lang">Javascript</li>
+                        <li class="lang">Ruby</li>
+                        <li class="lang">Html</li>
+                        <li class="lang">css</li>
                     </ul>
-                </address>
-            </header>
-        </div>
+                </div>    
+                <div class="box-container" id="frm-wrk-container">
+                    <div class="logo-container" id="frame-wrk-logo-container">
+                        <img src="images/frameworks.svg" alt="" class="type" id="frame-wrk-logo">
+                    </div>
+                    <div class="header-container">
+                        <h3 class="header">Framework</h3>
+                    </div>
+                    <ul class="container code-skill" id="frame-wrk-list">
+                        <li class="lang">Bootstrap</li>
+                        <li class="lang">Ruby on rails</li>
+                        <li class="lang">Rspec</li>
+                        <li class="lang">Copybare</li>
+                        <li class="lang">Selenium</li>
+                    </ul>
+                </div>    
```

Comment:
The framework should be styled as shown below
### Original design
![wrong](https://user-images.githubusercontent.com/58376020/126041002-b0307575-c432-4e3e-8d5a-fc0da9ae1a3c.png)

### Your Project
![wrong](https://user-images.githubusercontent.com/58376020/126041009-30d98cb6-0e0f-4525-9f96-e07239db1264.png)

Reply:
Done, I will submit now